code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -113,12 +113,13 @@
 	var/time_per_syringe = 250
 	var/last_synth = 0
 
-/obj/item/gun/syringe/chem/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
+/obj/item/gun/syringe/chem/attackby()

```

Comment:
no, we don't change override parameters because it propagates misinformation about the proc. Initialize and Destroy are the only exceptions